docs/errors/E207.md

```diff
@@ -0,0 +1,11 @@
+# E207: error escaped code point in unicode out of range
+
+The following code has escaped code point in unicode out of
+range.
+
+    let x = "hello\u{abcdef}";
+
+To fix this error, make sure that the escaped code point in
+unicode is less than 10FFFF.
```

Comment:
Nit: "less than 10FFFF" is incorrect. It should say "less than 110000" or "less than or equal to 10FFFF".

Nit: I'd phrase it this way:
```suggestion
To fix this error, make sure that the escaped code point is
between U+0000 and U+10FFFF inclusive.
```